docs/components/tabs.md

```diff
@@ -297,6 +268,43 @@ const tabs = ref<Tab[]>([
 </script>
 ```
 
+## Usage
+
+### Tab links
```

Comment:
I'm not sure I understand the new "Usage" section here -- can we just document "Tabs as Links" under the Props instead (above the Slots section)

Reply:
https://deploy-preview-2532--kongponents.netlify.app/components/tabs.html#tabs-as-links